src/Arcus.Scripting.Storage.FileShare/Arcus.Scripting.Storage.FileShare.psm1

```diff
@@ -29,3 +29,43 @@ function Create-AzFileShareStorageFolder {
 }
 
 Export-ModuleMember -Function Create-AzFileShareStorageFolder
+
+<#
+ .Synopsis
+  Upload a series of files at a given folder to a Azure File Share.
+
+ .Description
+  Upload a series of files at a given folder, matching an optional file mask, to a Azure File Share.
+
+ .Parameter ResourceGroupName
+  The resource group containing the Azure File Share.
+
+ .Parameter StorageAccountName
+  The Azure Storage account name that is hosting the Azure File Share.
```

Comment:
```suggestion
  The name of the Azure Storage account that is hosting the Azure File Share.
```

Just noticed that this was part of the Create-AzFileShareStorageFolder function as well. Could you update the description over there as well?

Reply:
Yes, done! Thanks for being thorough.